components/server/src/workspace/workspace-starter.ts

```diff
@@ -1370,11 +1370,11 @@ export class WorkspaceStarter {
             }
         }
 
-        let volumeSnapshotId = lastValidWorkspaceInstanceId;
-        // if this is snapshot or prebuild context, then try to find volume snapshot id in it
-        if (SnapshotContext.is(workspace.context) || WithPrebuild.is(workspace.context)) {
-            volumeSnapshotId = workspace.context.snapshotBucketId;
-        }
+        const volumeSnapshotId =
```

Comment:
Changing this to a constant, in hopes that we'll later be able to see the value [here](https://github.com/gitpod-io/gitpod/pull/12056/files#diff-b1c591e18af598ccba93b570ce7c97dece0988c9109a6d3be82bbe0df1fbd619R1383). Presently, we do not see the field at all in GCP logs. :thinking:

Reply:
This change has no effect at all. Just removes one of the rare comments. 

As commented [internally](https://gitpod.slack.com/archives/C02EN94AEPL/p1660283916694969?thread_ts=1660070394.317729&cid=C02EN94AEPL) the value is `undefined`, thus the property gets erased during serialization of the log item's payload, which is basically this:
```
> JSON.stringify({foo: undefined})
'{}'
```

Reply:
Thanks, @AlexTugarev !

We know `lastValidWorkspaceInstanceId` has a value (because it was logged in a separate field). 

So, now that we know fields with undefined values get removed from the logged object, we'll have to work to determine why `workspace.context.snapshotBucketId` is sometimes undefined.